common/types/src/gateway.rs

```diff
@@ -171,3 +171,25 @@ impl fmt::Display for GatewayIpPacketRouterDetails {
         writeln!(f, "\taddress: {}", self.address)
     }
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GatewayWireguardDetails {
```

Comment:
At some point I wanted to include it in the gateway details, but since it's now only in nym-node, maybe it doesn't make sense anymore. If we want to include it later, we can re-add it, for now I'll just delete this code.